Nitrox.Bootloader/Main.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Nitrox.Bootloader
+{
+    public static class Main
+    {
+        private static readonly Lazy<string> nitroxLauncherDir = new Lazy<string>(() =>
+        {
+            string nitroxAppData = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Nitrox");
+            if (!Directory.Exists(nitroxAppData))
+            {
+                return null;
+            }
+            string nitroxLauncherPathFile = Path.Combine(nitroxAppData, "launcherpath.txt");
+            if (!File.Exists(nitroxLauncherPathFile))
+            {
+                return null;
+            }
+
+            try
+            {
+                string valueInFile = File.ReadAllText(nitroxLauncherPathFile).Trim();
+                Task.Factory.StartNew(() =>
+                {
+                    try
+                    {
+                        // Delete the path so that the launcher should be used to launch Nitrox
+                        File.Delete(nitroxLauncherPathFile);
+                    }
+                    catch (Exception)
+                    {
+                        // ignored
```

Comment:
Why not implement a exception just in case. Would be useful for devs in `DEBUG` if something is causing an error.

Reply:
It won't be useful. Nitrox won't run so reason enough to investigate.